src/Vocab/IVocab.cs

```diff
@@ -2,14 +2,22 @@
 
 public interface IVocab
 {
-    string GetUnknownValue();
+
     Dictionary<string, long> Values { get; }
     Dictionary<long, string> Indices { get; }
     Dictionary<string, long> SpecialValues { get; }
     Dictionary<long, string> SpecialIndices { get; }
 
     IEnumerable<string> SpecialTokens();
 
+    string GetUnknownValue();
+    string GetBosValue();
```

Comment:
We can remove these methods if it's not needed anymore after the other comment.

Reply:
They are still in use for a few test cases; I will ensure to remove them later if they become unnecessary.